generate_grpc.sh

```diff
@@ -17,6 +17,7 @@ buf lint
 protoc -I proto/ptypes/ -I proto/vendor/protobuf/src/ proto/ptypes/common.proto --go_out=plugins=grpc:proto/ptypes/
 python3 -m grpc_tools.protoc -I proto/ptypes/ -I proto/vendor/protobuf/src/ --python_out=integration/rpc/controller --grpc_python_out=integration/rpc/controller proto/ptypes/common.proto
 python3 -m grpc_tools.protoc -I proto/ptypes/ -I proto/vendor/protobuf/src/ --python_out=integration/rpc/replica --grpc_python_out=integration/rpc/replica proto/ptypes/common.proto
+python3 -m grpc_tools.protoc -I proto/ptypes/ -I proto/vendor/protobuf/src/ --python_out=integration/rpc/sync --grpc_python_out=integration/rpc/sync proto/ptypes/common.proto
```

Comment:
One question: Do we need to switch to the format of the instance-manager grpc generate script for this repo?

Reply:
It would be good to do so, though there isn't a conflict yet that would require it. I would prefer to do that work in a separate issue. If you agree, I can create it.